nixos/doc/manual/default.nix

```diff
@@ -72,6 +73,63 @@ let
       </toc>
     '';
 
+  manualXsltprocOptions = toString [
```

Comment:
These changes sounds like an issue that should be fixed any way, can you open another pull request to get these changes merged independently of the taskserver modification.